playwright_stealth/js/navigator.userAgent.js

```diff
@@ -1,5 +1,5 @@
 // replace Headless references in default useragent
 const current_ua = navigator.userAgent
 Object.defineProperty(Object.getPrototypeOf(navigator), 'userAgent', {
-    get: () => opts.navigator_user_agent || current_ua.replace('HeadlessChrome/', 'Chrome/')
+    get: () => opts.navigator_user_agent || current_ua.replace(/Headless/g, '')
```

Comment:
There is no check if opts is undefined (also in the original code, yes). This will break scripts on other websites that rely on navigator.userAgent